release_notes/ocp_3_11_release_notes.adoc

```diff
@@ -169,6 +169,13 @@ the tools curated by the Kubernetes upstream development community.
 Functionality is tailored toward ISVs and development teams, versus ops teams.
 The most advanced Operators that exist are using Golang.
 
+[[ocp-311-brokers]]
+=== Brokers
+
```

Comment:
Hey @tmorriso-rh I think that we should have a few subsections under the Brokers per the What's New presentation: one heading on the Galaxy integration (with details on that feature), then one on broker support for authenticated registries, then another on the service catalog access control.

Reply:
Thanks, I added those subsections.